cypress/integration/price/pricelist_with_currency_CHF_and_precision_3_spec.js

```diff
@@ -0,0 +1,33 @@
+import { PriceList } from '../../support/utils/pricelist';
+import { Pricesystem } from '../../support/utils/pricesystem';
+import { humanReadableNow } from '../../support/utils/utils';
+
+describe('Create a Pricelist with currency CHF and precision 3: https://github.com/metasfresh/metasfresh-e2e/issues/195', function() {
```

Comment:
Could be just my nitpicking here, but is there a reason for us to add the issue link since it's all stored inside the git history?

@cristinastefan90 and @teosarca could you give your input here?

Reply:
I think this was the approach at the beginning of the cypress tests, because I have also seen such a link in some older ones. I would say it makes no sense to have it here, a short description is enough as we have it all in git.